libs/auth/src/signed_hash_validation.ts

```diff
@@ -1,95 +1,123 @@
+import { existsSync } from 'fs';
```

Comment:
This commit is easier to review using https://github.com/votingworks/vxsuite/pull/5302/commits/47907c2e118621979f62e9a994f79cbbcd039882?w=1

Also, this is probably the only commit that you'd want to review @benadida

Reply:
Also FYI @benadida that I plan to update https://check.voting.works in backwards compatible way so that it works for both v3.1 and v4.0

Reply:
that's amazing, thank you.